docs/real-time-analytics/discounting/process.sql

```diff
@@ -0,0 +1,66 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create Discount Code Table
+CREATE STREAM DISCOUNT_CODES
+(
+    CODE VARCHAR PRIMARY KEY, 
+    PERCENTAGE DOUBLE
+)
+WITH (
```

Comment:
For consistency, please pull the `)` into this line (and delete previous line

```suggestion
) WITH (
```